pkg/executor/cte.go

```diff
@@ -553,18 +570,6 @@ func (p *cteProducer) resetTracker() {
 	}
 }
 
-func (p *cteProducer) reopenTbls() (err error) {
-	if p.isDistinct {
-		p.hashTbl = join.NewConcurrentMapHashTable()
-	}
```

Comment:
Is this useless?

Reply:
Yes, after `reopenTbls`, this producer will be opened again, and the `hasTbl` will be created in `openProducerExecutor`